packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -101,11 +100,18 @@ export default function QueryInspectorControls( props ) {
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
 	const showInheritControl = isControlAllowed( allowedControls, 'inherit' );
+	const inheritControlLabel = __( 'Query Type' );
+	const inheritControlDefaultHelp = __(
+		'Display a list of posts or custom post types based on the current template.'
+	);
+	const inheritControlCustomHelp = __(
+		'Display a list of posts or custom post types based on specific criteria.'
+	);
```

Comment:
I think it'd be fine to not use these as constants, but apply the labels in the controls directly. We're not reusing any of these anyhow and it makes the code a bit more readable for those less familiar.